python_modules/dagster/dagster/daemon/cli/__init__.py

```diff
@@ -39,7 +40,7 @@ def run_command():
             _daemon_run_command(instance)
 
 
-@telemetry_wrapper
+@telemetry_wrapper(metadata={"DAEMON_SESSION_ID": get_telemetry_daemon_session_id()})
```

Comment:
might be good to move to an explicit log_action call and explicit setting of the global to a value here when we know we are running a daemon